### ISSUE TITLE
HTTP server libraries

### ISSUE THREAD

**Issue by chris-martin** [haskell-lang/haskell-lang#106]
I think the libraries page could use something related to web servers - should Warp be on the list?

**Comment by snoyberg** [comment]
Certainly sounds good to me. You interested in taking a stab at such content?

**Comment by chris-martin** [comment]
Sure, I can. Does anything else in that category belong on the list? And is Warp the best WAI-related package to highlight?

**Comment by psibi** [comment]
I think they should belong on the list: http://www.yesodweb.com/book/deploying-your-webapp#deploying-your-webapp_warp

But I guess only Warp is for production systems.

**Comment by snoyberg** [comment]
Yeah, I wouldn't mention the others. CGI/FastCGI/SCGI are pretty niche
these days.
